components/camel-jms/src/main/java/org/apache/camel/component/jms/JmsConfiguration.java

```diff
@@ -479,7 +479,10 @@
 
     @UriParam(label = "producer", description = "Sets whether JMS date properties should be formatted according to the ISO 8601 standard.")
     private boolean formatDateHeadersToIso8601;
-
+    
+    @UriParam(label = "deliveryDelay", description = "set delivey delay for jms")
```

Comment:
In current latest commit is still wrong label - it should be "producer" instead of "deliveryDelay". 

Description should contain JMS 2.0 warning. I suggest this: "This option requires JMS 2.0 compliant broker."

Reply:
ok. can you check it?